docs/src/components/CurrentRefinements.md

```diff
@@ -16,26 +16,51 @@ Show the currently refined values and allow them to be unset.
 ## Usage
 
 ```html
-<ais-current-refinements :clears-query="true"></ais-current-refinements>
+<template>
+  <ais-current-refinements
+    :excluded-attributes="[]"
+    :transform-items="transformItems"
+  ></ais-current-refinements>
+</template>
+<script>
+export default {
+  data() {
+    return {
+      transformItems: items =>
+        items.map(item => {
+          item.label = item.label.toLocaleUpperCase();
+          return item;
+        })
+    };
+  }
+};
+</script>
 ```
 
 ## Props
 
 Name | Type | Default | Description | Required
 ---|---|---|---|---
-transformItems | Function | | Allows you to format and change the attributes | -
-clearsQuery | Boolean | `false` | Should the 'clear all' button also clear the query? | -
-excludedAttributes | Array | `[]` | Attributes not to show or clear | -
+includedAttributes | Array | | Attributes to show or clear | -
+excludedAttributes | Array | `["query"]` | Attributes not to show or clear | -
 transformItems | `(items: object[]) => object[]` | `x => x` | Function which receives the items, which will be called before displaying them. Should return a new array with the same shape as the original array. Useful for mapping over the items to transform, remove or reorder them | -
 classNames | Object | | Override class names | no
 
+Note that you can not use `includedAttributes` and `excludedAttributes` at the same time. Included attributes are exclusive, and thus take precedence over excluded attributes.
+
 ## Slots
 
 name | scope | Description
 ---|---|---
-`default` | `{ items: Item[], refine: Item => void, clearAll: () => void }` | Override how all the items look
+`default` | `{ items: Item[], refine: Item => void }` | Override how all the items look
 `item` | `{ item: Item, refine: Item => void }` | Override how an item looks
-`clearAllLabel` | `{ items: Item[] }` | Override how the "clear all" button looks
+
+an Item has the keys:
```

Comment:
```suggestion
An item has the keys:
```

Reply:
On purpose, the name of the type is `Item`. Will put it in backticks